### DISCUSSION TITLE
add notification for incoming call and background mode. [капец]

### DISCUSSION THREAD

**Discussion by delasty709** [stoatchat/discussions#496] [Feature Suggestions]
I really miss the incoming call notification, when they call you, you don’t see it and for this you miss the call 
.
где увидомление при входящем звонке? и фонового режима нету жесть
дискорд гавно полное, думал эта прога заменит его, это почти свершилось но и за отсутствие: демки, работа на фоне, и уважения при входящем звонке, пользоваться не возможно, но прога рил ахуеная, достойна уважения, у неё есть шанс выебать дс :з

**Comment by Zomatree** [comment]
Revolt is still in beta and lots of the features are incomplete and not implemented yet. The voice system is being completely rewritten so expect most of the current issues with it to be gone eventually, as for background mode, revolt minimizes to tray and still works in the background.

**Reply by delasty709** [reply]
Thank you very much for your answer, I will wait for new achievements)
